src/main/java/com/example/management/models/Movie.java

```diff
@@ -0,0 +1,95 @@
+package com.example.management.models;
+
+import javax.persistence.*;
+import java.util.*;
+/**
+ * Movie
+ */
+@Entity
+@Table(name = "movie")
+public class Movie extends BaseEntity{
+
+    private String name;
+    private String description;
+    private int year;
+    private Date releaseDay;
+    private String image;
+
+    @ManyToMany

```

Comment:
I think in this case, because this is an n-n relationship, if a movie_type is deleted then a movie with that type should be deleted as well. But the movie belongs to many types and the type attribute can be null, so we don't have to delete it. I wonder is it right to use cascadeType.SAVE_UPDATE in this case?